raiden_contracts/tests/fixtures/service_registry_fixtures.py

```diff
@@ -3,11 +3,48 @@
 import pytest
 from web3.contract import Contract
 
-from raiden_contracts.constants import CONTRACT_SERVICE_REGISTRY
+from raiden_contracts.constants import CONTRACT_SERVICE_REGISTRY, EMPTY_ADDRESS
+from raiden_contracts.tests.utils import (
+    CONTRACT_DEPLOYER_ADDRESS,
+    DEFAULT_BUMP_DENOMINATOR,
+    DEFAULT_BUMP_NUMERATOR,
+    DEFAULT_DECAY_CONSTANT,
+    DEFAULT_MIN_PRICE,
+    DEFAULT_REGISTRATION_DURATION,
+)
 
 
 @pytest.fixture(scope="session")
 def service_registry(deploy_tester_contract: Callable, custom_token: Contract) -> Contract:
     return deploy_tester_contract(
-        CONTRACT_SERVICE_REGISTRY, [custom_token.address, 3000 * (10 ** 18)]
+        CONTRACT_SERVICE_REGISTRY,
+        [
+            custom_token.address,
+            CONTRACT_DEPLOYER_ADDRESS,
+            3000 * (10 ** 18),
```

Comment:
I usually prefer to write these as `int(3000e18)`, but that's a matter of taste.

Reply:
I'm not a seasoned Python user so I follow your taste here.